src/Context/wishlistContext.js

```diff
@@ -0,0 +1,37 @@
+import { createContext, useContext, useReducer } from "react";
+
+const WishlistContext = createContext();
+
+const wishFunction = (wishState, action) =>{
+    switch(action.type){
+        case "Add_to_wishlist" : 
```

Comment:
Good practice is using capital letters for cases like "ADD_TO_WISHLIST". You can change that later.

Reply:
Okay